aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -4,53 +4,60 @@
 use crate::types::{AtomicTxnIndex, MVAggregatorsError, TxnIndex};
 use aptos_aggregator::{
     aggregator_change_set::{AggregatorApplyChange, ApplyBase},
-    types::{AggregatorID, AggregatorValue},
+    types::AggregatorValue,
 };
 use claims::{assert_matches, assert_none};
 use crossbeam::utils::CachePadded;
 use dashmap::DashMap;
 use std::{
     collections::btree_map::{BTreeMap, Entry},
+    fmt::Debug,
+    hash::Hash,
     iter::DoubleEndedIterator,
     sync::atomic::Ordering,
 };
 
+pub enum CommitError {
+    CodeInvariantError(String),
```

Comment:
Why String? Shouldn't it be some StatusCode?

Reply:
We cannot do anything different for different CodeInvariantError, so seems best to have them describe what they want, so we can print that out, and do what we do for all CodeInvariantErrors the same